bothan-core/src/manager/crypto_asset_info/price.rs

```diff
@@ -0,0 +1,18 @@
+use crate::manager::crypto_asset_info::price::tasks::get_signal_price_states;
+use crate::manager::crypto_asset_info::types::{PriceState, WorkerMap};
+use crate::registry::{Registry, Valid};
+
+pub(crate) mod cache;
+pub(crate) mod tasks;
+
+pub async fn get_prices<'a>(
```

Comment:
i dont' think this function is necessary? why not just do this logic in `CryptoAssetInfoManager::get_prices`?